gradle.properties

```diff
@@ -1,12 +1,12 @@
 projectVersion=2.5.1-SNAPSHOT
 micronautDocsVersion=2.0.0.RC1
 micronautBuildVersion=1.1.5
-micronautVersion=3.0.0-SNAPSHOT
-micronautTestVersion=2.3.7
+micronautVersion=3.0.0-M5
+micronautTestVersion=
 micronautR2dbcVersion=1.1.1
```

Comment:
There is a 2.0 version of this module out https://search.maven.org/artifact/io.micronaut.r2dbc/micronaut-r2dbc-core/2.0.0-RC2/jar